project/AppDependencies.scala

```diff
@@ -2,21 +2,18 @@ import sbt.*
 
```

Comment:
`import sbt.*` can be replaced by `import sbt._`

Reply:
Discussed about this may be helpful to leave for Scala 3